test/helpers.py

```diff
@@ -54,3 +55,25 @@ def deploy_app_externally(
         print(f"Deploying app failed!\n### stdout ###\n{stdout_s}\n### stderr ###\n{stderr_s}")
         raise Exception("Test helper failed to deploy app")
     return stdout_s
+
+
+class PopenWithCtrlC(subprocess.Popen):
```

Comment:
We have an identical helper in the tests of synchronicity, but it's not part of the synchronicity *package*, so we can't reuse it here